server/weather/weatherHandler.js

```diff
@@ -1,28 +1,35 @@
-'use strict'
+const boom = require('boom');
+const httpStatus = require('http-status');
 
-const boom = require('boom')
-const httpStatus = require('http-status')
+const weatherCtrl = require('./weatherCtrl');
+const logger = require('../utils/logger');
 
-const weatherCtrl = require('./weatherCtrl')
-const logger = require('../utils/logger')
+const getWeatherByCityName = async (req, res) => {
+	const { cityName } = req.query;
 
-const getWeatherByCityName = async function (req, res) {
-  const cityName = req.query.cityName
+	try {
+		const data = await weatherCtrl.getWeatherByCityName(cityName);
 
-  try {
-    const data = await weatherCtrl.getWeatherByCityName(cityName)
-    return res({
-      name: data.name,
-      coord: data.coord,
-      weather: data.weather
-    })
-  } catch (error) {
-    const errorMessage = `Failed to fetch weather for ${cityName}`
-    !error.logged && logger.error(error, errorMessage)
-    return res(boom.boomify(error, { statusCode: httpStatus.INTERNAL_SERVER_ERROR, message: errorMessage }))
-  }
-}
+		return res({
```

Comment:
Spaces seem to occur in these files [1]. Can you check your editor ?

Reply:
no issue with editor, it is set to tabs